### ISSUE TITLE
Standard board size seems random and can't be changes

### ISSUE THREAD

**Issue by eladhen** [lucasnlm/antimine-android#317]
**Describe the bug**
I used to have a 18*10 board on Standard level with no way to adjust. This size fit my screen so it was fine. After the last update (14.0.0) standard size changed to 21*13, much larger and doesn't fit my screen unless I zoom out. 

**To Reproduce**
Notsure, because I'm not sure how the game decides the board size on standard.

**Expected behavior**
I'd like to be able to customize standard lecvel board size to fit my screen.

**Android Version:**
10

**App Version**
14.0.0

**Comment by lucasnlm** [comment]
On `14.0.0` Standard difficulty will increase size of minefield to keep a maximum ratio of mines/size.
Before this change, it was possible to have a Standard with a mines ratio bigger than "Expert" level.
So, depending on how many Standard games have you won, the Standard size will be larger than your screen size.

What I can do is create a new fixed size level. What do you think about it?

**Comment by eladhen** [comment]
Yes, I would like to have a fixed size version! That's what I enjoy playing. I really loved the "normal" setting of the game before it changed size. I really love this version of minesweeper. Thank you!

**Comment by lucasnlm** [comment]
Thanks again for your feedback!
I will add fixed size level on the next update.

**Comment by lucasnlm** [comment]
<img src="https://user-images.githubusercontent.com/1326126/132427771-b292a148-7c16-41e1-b839-941cdef9e225.png" width="300px" />

On the next update, "Standard" will be renamed to "Progressive" and a new "Fixed Size" difficulty will be available.

**Comment by eladhen** [comment]
Will it be possible to change the size of "fixed size" manually?

**Comment by lucasnlm** [comment]
@eladhen Currently not. The current "fixed size" is calculated to fill the device screen.
But you can create a new game with a different fixed size using "Custom" mode.

**Comment by eladhen** [comment]
The fixed size in version 14.2 doesn't add mines after each game. What I want is a mode like "normal" in version 12...

**Comment by lucasnlm** [comment]
@eladhen I can try do something about it, but add more mines with fixed size will create impossible games (usually after 22% of the area)

**Comment by eladhen** [comment]
I understand that, but the dynamic of going up and down the stages in this mode is exactly what I want. I reinstalled version 12 to play normal mode. If the level is impossible you fale and drop one mine. It's fine and fun.

**Comment by lucasnlm** [comment]
I think if I add an option on "gameplay" Settings to enable/disable this feature, it may help you. Is it ok to you?

**Comment by eladhen** [comment]
I feel like that's hiding the game's best feature, but if it's available, I'll be happy.

**Comment by lucasnlm** [comment]
@eladhen The increase of mines in "Fixed Size" mode will return in "15.0.0" but with a new system to avoid impossible levels. For example: in a Fixed Size level it will try place X mines, but if it fails to create a solvable level it will remove some of them (especially to ensure the first clicked are is safe).
